lightly/utils/lars.py

```diff
@@ -0,0 +1,158 @@
+import torch
+from torch.optim.optimizer import Optimizer, required
+
+
+class LARS(Optimizer):
```

Comment:
Had to add this because PyTorch Lightning Bolts is not compatible with newer PyTorch Lightning versions. It is also the only dependency we had from PyTorch Lightning Bolts, so it is easier to just maintain it ourselves instead of adding an additional dependency.